docs/site/tutorials/protocol_oriented_generics.ipynb

```diff
@@ -0,0 +1,1141 @@
+{
```

Comment:
<p>Given that a popular use of inheritance is to represent an "is-a" relationship, inheriting from both House and Lawn looks surprising, since it's hard to imagine something that is both a house and a lawn. (Same for a house and an apartment from the code snippets above).</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/swift/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>